br/pkg/task/restore.go

```diff
@@ -277,6 +278,8 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 		TTL:      utils.DefaultBRGCSafePointTTL,
 		ID:       utils.MakeSafePointID(),
 	}
+	g.Record("BackupTS", restoreTS)
```

Comment:
I guess `backupMeta.Version` would be more suitable here?

Also cc @3pointer

Reply:
Juncen, luancheng and me have talked about this, restore and backup as one atom task in tidb, if split brie into 2 tasks, it take too much effort, big impact in tidb, so we would like to keep this change as it.

Reply:
OK